ports/unix/Makefile

```diff
@@ -1,3 +1,8 @@
+# Fail fast when pkg-config is not installed
+ifeq ($(shell which pkg-config || echo 0), 0)
```

Comment:
`pkg-config` is only required for specific configurations (not `MICROPY_STANDALONE`  or `MICROPY_BLUETOOTH_BTSTACK` so this check should probably be guarded by the same conditions.